packages/cli/test/helpers/stubs/sni-endpoints.ts

```diff
@@ -1,109 +1,115 @@
-export type Endpoint = {
-  name: string
-  cname: string | null
-  display_name?: string
-  domains?: string[]
-  ssl_cert: {
-    'ca_signed?': boolean
-    cert_domains: string[]
-    starts_at: string
-    expires_at: string
-    issuer: string
-    subject: string
-    acm?: boolean
-  }
-}
+import {SniEndpoint} from '../../../src/lib/types/sni_endpoint'
+import {Domain} from '../../../src/lib/types/domain'
 
-export const endpointStables: Endpoint = {
+export const endpointStables: Partial<SniEndpoint> = {
```

Comment:
This type looks more like `Pick<SniEndpoint, 'name' | 'domains' | 'ssl_cert'>` rather than `Partial`